frontend/src/Metamaps/Visualize.js

```diff
@@ -152,8 +152,9 @@ const Visualize = {
 
     // monkey patch scale function
     const oldScale = self.mGraph.canvas.scale
+    const cachedPathname = window.location.pathname
     const updateScaleInUrl = debounce(() => {
-      Util.updateQueryParams({ scale: self.mGraph.canvas.scaleOffsetX.toFixed(2) })
+      Util.updateQueryParams({ scale: self.mGraph.canvas.scaleOffsetX.toFixed(2) }, pathname)
```

Comment:
thanks, fixed. it was working anyways since I had a default argument in Util, but fixing is good